terraform/modules/web/main.tf

```diff
@@ -0,0 +1,85 @@
+locals {
+  domain_prefix = var.staging ? "stage.${var.app}" : "${var.app}"
+}
+
+data "aws_acm_certificate" "this" {
+  domain   = "${local.domain_prefix}.cms.gov"
+  statuses = ["ISSUED"]
+}
+
+data "aws_wafv2_web_acl" "this" {
+  name  = "SamQuickACLEnforcingV2"
+  scope = "CLOUDFRONT"
+}
```

Comment:
Rather than rely on the SamQuickACLEnforcingV2 ACL, this module should create and manage a separate ACL and associated rules for each distribution.